packages/cli/src/zosfiles/compare/CompareBaseHelper.ts

```diff
@@ -0,0 +1,168 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { IHandlerParameters, DiffUtils } from "@zowe/imperative";
+import {  IZosFilesResponse } from "@zowe/zos-files-for-zowe-sdk";
+import { ICompareFileOptions } from "./doc/ICompareFileOptions";
+
+/**
+ * Base helper for compare command to handle common operations through the comparison
+ * @export
+ */
+export default class CompareBaseHelper {
```

Comment:
Please remove the `default` keyword. I think we generally prefer to avoid using it, and use explicitly named imports/exports instead, except in cases where it is required like for command handler files.